docs/get-started/installation/prepare/orange-pi-3-lts.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Orange Pi 3 LTS Setup Guide"
+linkTitle: "Orange Pi 3 LTS Setup"
+weight: 16
+type: "docs"
+description: "Image an Orange Pi 3 LTS to prepare it for viam-server installation."
+images: ["/get-started/installation/thumbnails/orange-pi-3-LTS.png"]
+imageAlt: "Orange Pi 3 LTS"
+no_list: true
+# SME: Olivia Miller
+---
+
+The [Orange Pi 3 LTS](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) is a highly compact open-source single-board computer equipped with dual-band wireless WiFi and Bluetooth 5.0.
```

Comment:
nit: Wifi and Bluetooth are both wireless by design.

```suggestion
The [Orange Pi 3 LTS](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) is a highly compact open-source single-board computer equipped with dual-band WiFi and Bluetooth 5.0.
```

Reply:
Lol true